pyocd/probe/pydapaccess/interface/__init__.py

```diff
@@ -59,6 +59,7 @@
     elif system == "Linux":
         USB_BACKEND = "pyusb"
     else:
-        raise DAPAccessIntf.DeviceError("No USB backend found")
+        #raise DAPAccessIntf.DeviceError("No USB backend found")
```

Comment:
Should check the `PyUSB.isAvailable` class attribute first. If that is False, then raise the exception, otherwise set the backend and continue.